package.json

```diff
@@ -74,33 +78,33 @@
     "ember-cli-sass": "^11.0.1",
     "ember-concurrency": "^3.1.1",
     "ember-config-helper": "^0.1.3",
-    "ember-data": "~5.3.3",
+    "ember-data": "~5.3.8",
+    "ember-data-table": "^2.1.0",
```

Comment:
This is needed because Appuniversum no longer uses this addon but we do and we didn't list it as a dependency (which only worked by accident because Appuniversum brought it in).